Examples/SearchExamples/SimpleUISearchViewController.swift

```diff
@@ -1,12 +1,19 @@
 import MapboxMaps
 import MapboxSearch
 import MapboxSearchUI
+import MapKit
 import UIKit
 
 class SimpleUISearchViewController: MapsViewController {
     lazy var searchController: MapboxSearchController = {
         let locationProvider = PointLocationProvider(coordinate: .sanFrancisco)
-        var configuration = Configuration(locationProvider: locationProvider)
+        let formatter = MKDistanceFormatter()
```

Comment:
Maybe https://developer.apple.com/documentation/foundation/measurementformatter instead? MapKit is our direct competitor😆